va-gov/pages/pension/index.md

```diff
@@ -152,12 +152,12 @@ VA pension benefits provide tax-free monthly payments to wartime Veterans with l
   <h2>More Information and Resources</h2>
```

Comment:
The links for "Check Your VA Claim or Appeal Status", "View Your VA Payment History", and "Change Your VA Direct Deposit and Contact Information" are now `/claim-or-appeal-status/`, `/va-payment-history/, and `/change-direct-deposit-and-contact-information/` according to the spreadsheet.

Reply:
I only caught it cuz I just caught it on myself haha